mlir/include/mlir/Dialect/Quant/QuantTypes.h

```diff
@@ -397,11 +401,12 @@ class UniformQuantizedPerAxisType
 ///
 /// Syntax synopsis:
 ///   Per-layer, all parameters expressed:
-///     !quant<quantile[StorageType:ExpressedType]{Quantiles}:{Scale:ZeroPoint}>
+///     !quant<quantile[StorageType:QuantileType:ExpressedType]{Quantiles}:{Scale:ZeroPoint}>
 ///   Per-layer, optional parameters omitted:
-///     !quant<quantile[StorageType]{Quantiles}:{Scale}>
+///     !quant<quantile[StorageType:QuantileType]{Quantiles}:{Scale}>
 ///
 ///   StorageType: 'i'|'u' NumBits
+///   QuantileType: 'i'|'u' NumBits, 'hf8', 'bf8', 'f16', 'bf16', 'f32', 'f64'
```

Comment:
It would be nice to add a little description of what this means.

Reply:
sure, I will add it: I tried to replicate the pre-existing style for the description, which is a bit cryptic

Reply:
Regarding check-mlir: I've run mlir-opt compiled with our build system and using FileCheck to check the mlir tests for the quant dialect. I'll run also `check-mlir`.

Reply:
The report from check-mlir seems ok:

![image](https://github.com/user-attachments/assets/6a1f9812-7f1a-4843-89dc-e0aec39a0770)

Reply:
Added more verbose comment about what types are supported for quantileType member